lms/static/scripts/frontend_apps/components/test/AppRoot-test.js

```diff
@@ -0,0 +1,78 @@
+import { mount } from 'enzyme';
+
+import mockImportedComponents from '../../../test-util/mock-imported-components';
+import { useConfig } from '../../config';
+import { useService } from '../../services';
+import AppRoot, { $imports } from '../AppRoot';
+
+describe('AppRoot', () => {
+  beforeEach(() => {
+    $imports.$mock(mockImportedComponents());
+
+    // Suppress warning when exceptions are thrown when rendering.
+    sinon.stub(console, 'warn');
```

Comment:
This can be removed once the Babel issue is resolved. See https://github.com/hypothesis/frontend-shared/pull/868.